arkouda/client.py

```diff
@@ -40,9 +40,7 @@
 maxTransferBytesDefVal = 2**30
 maxTransferBytes = maxTransferBytesDefVal
 AllSymbols = "__AllSymbols__"
-EmptySymbolTable = "__EMPTY_SYMBOLTABLE__"
 RegisteredSymbols = "__RegisteredSymbols__"
```

Comment:
If folks are good with this approach it might not be a bad time to pull over AllSymbols and RegisteredSymbols with the other two.

Reply:
That's a good idea! It makes more sense for `AllSymbols` and `RegisteredSysmbols` to be in `infoclass.py`. I'll go ahead and do that